src/testrender/simpleraytracer.cpp

```diff
@@ -1077,14 +1149,15 @@ SimpleRaytracer::subpixel_radiance(float x, float y, Sampler& sampler,
     return path_radiance;
 }
 
-Color3
+
+OSL_HOSTDEVICE Color3
 SimpleRaytracer::antialias_pixel(int x, int y, ShadingContext* ctx)
 {
     Color3 result(0, 0, 0);
     for (int si = 0, n = aa * aa; si < n; si++) {
         Sampler sampler(x, y, si);
         // jitter pixel coordinate [0,1)^2
-        Vec3 j = sampler.get();
+        Vec3 j = no_jitter ? Vec3(0, 0, 0) : sampler.get();
```

Comment:
I think you want 0.5f, 0.5f in the `no_jitter` case ? Otherwise you always get -1 from the warp below.

Reply:
Ah, good catch.